### PR TITLE
Status bar visibility

### PR DESCRIPTION
**Pull request by marcosrdz**

### CODE REVIEW

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/9e8v6dv823adzvd4thtg31e5g0

[download apk](http://filestorage.bluewallet.io:1488/1591148459.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/5rxqauhg171769zx77ewyctz08

[download apk](http://filestorage.bluewallet.io:1488/1591150679.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/4f0guv6jwnbtc9mzpbgjnzc2nw

[download apk](http://filestorage.bluewallet.io:1488/1591210043.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/yv4nu8q65xfefb3a4zmz5rtv2w

[download apk](http://filestorage.bluewallet.io:1488/1591275525.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/29ug47nn13uayfj17y8rgnv960

[download apk](http://filestorage.bluewallet.io:1488/1591292363.apk)

File: BlueComponents.js [hunk 1]
```diff
@@ -1903,9 +1896,7 @@ const WalletCarouselItem = ({ item, index, onPress, handleLongPress }) => {
           onPressOut={onPressedOut}
           onLongPress={handleLongPress}
           onPress={() => {
-            onPressedOut();
```
Comment (Overtorment) [inline BlueComponents.js:1906]: I dont understand this but I hope you know what you're doing

File: ios/BlueWallet/Info.plist [hunk 2]
```diff
@@ -94,7 +94,7 @@
 	<key>NSCalendarsUsageDescription</key>
 	<string>This alert should not show up as we do not require this data</string>
 	<key>NSCameraUsageDescription</key>
-	<string>In order to quickly scan the recipient's address, we need your permission to use the camera to scan their QR Code.</string>
+	<string>In order to quickly scan the recipient&apos;s address, we need your permission to use the camera to scan their QR Code.</string>
```
Comment (Overtorment) [inline ios/BlueWallet/Info.plist:97]: not the first time I'm seeing `'` changes to `&apos;` and back in plist. can we finally settle on something and stop?

File: BlueComponents.js (hunk 1 above)
Comment (marcosrdz) [reply BlueComponents.js:1906]: onPressout prop was calling it. It was redundant to also call it in onPress

File: ios/BlueWallet/Info.plist (hunk 2 above)
Comment (marcosrdz) [reply ios/BlueWallet/Info.plist:97]: lint auto fix  changes this on my end with this result.
Idk if previous Changes were with a different tool

**Pull request merged**